vtr_flow/scripts/run_vtr_flow.py

```diff
@@ -525,6 +525,8 @@ def vtr_command_main(arg_list, prog=None):
 
     assert args.temp_dir
     temp_dir = Path(args.temp_dir)
+    if not str(temp_dir).startswith("/"):
```

Comment:
Does parmys still need an absolute path?  If not, we can remove this line I think.  If so, this should be commented (say why we're making an absolute path).

Reply:
As far as I can tell, parmys still fails when an absolute path to the temp folder is not included. I have added a comment to this if statement mentioning this.